app/src/main/java/com/cobresun/factfulnewsandroid/TabFragment.kt

```diff
@@ -3,12 +3,12 @@ package com.cobresun.factfulnewsandroid
 
 import android.net.Uri
 import android.os.Bundle
-import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.browser.customtabs.CustomTabsIntent
 import androidx.core.content.res.ResourcesCompat
+import androidx.fragment.app.Fragment
```

Comment:
Did you move where this import was? LMAO

Reply:
oh yea idk why